kubesignin/values.yaml

```diff
@@ -16,39 +16,12 @@ Dex:
 
   # # Must set your own credentials.
   # # Use base64 encoded value.
-  # Connector:
-    # MockCallback: true
-    # GitHub:
-    #   Name: "GitHub"
+  Connectors:
+    GitHub: []
+    # - Name: "GitHub"
```

Comment:
these are the default values of the package, but the different github connectors should be provided in each installation, the comments here are only an example to indicate how to provide those values.